.changeset/rotten-lemons-provide.md

```diff
@@ -0,0 +1,6 @@
+---
+'@nordeck/matrix-neoboard-widget': major
```

Comment:
is it?

Reply:
Well react-scripts to vite? Not sure.

Reply:
the react-sdk I understand, but the widget I'm not so sure